src/qml/ProjectCreationScreen.qml

```diff
@@ -0,0 +1,346 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import QtQuick.Controls.Material.impl
+import org.qfield
+import Theme
+
+Page {
+  id: createNewProject
+
+  signal create(var projectConfig)
+
+  leftPadding: mainWindow.sceneLeftMargin
+  rightPadding: mainWindow.sceneRightMargin
+
+  header: QfPageHeader {
+    title: qsTr("Create a new project")
+
+    backgroundFill: false
+    showBackButton: true
+    showApplyButton: false
+    showCancelButton: false
+    showMenuButton: false
+
+    topMargin: mainWindow.sceneTopMargin
+    titleColor: Theme.mainTextColor
+    titleFont: Theme.strongTitleFont
+
+    onBack: {
+      createNewProject.visible = false;
+    }
+  }
+
+  Flickable {
+    anchors.top: parent.top
+    anchors.left: parent.left
+    anchors.right: parent.right
+    anchors.bottom: createProjectButton.top
+    anchors.margins: 16
+    contentHeight: newProjectConfigColumn.height
+    clip: true
+
+    Column {
+      id: newProjectConfigColumn
+      width: parent.width
+      spacing: 8
+
+      Label {
+        text: qsTr("Project Name")
```

Comment:
We don't really need this, we can use the placeholder + a default project name in the text field itself. Instead, let's use it as a micro introduction. 

```suggestion
        text: qsTr("This page will guide you through the creation of a simple project through simple configuration toggles. For more advanced needs, we advise you to create projects in QGIS using QFieldSync.")
```